From cb3ff8e9835c72dcfacce65e283c3042c340f4a6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Tue, 23 Jan 2024 10:18:37 +0100
Subject: [PATCH] menu: Fix hiding the arrow

It fixes the problem that the whole image become hidden instead of the arrow when the "show-arrow" setting is changed.
---
 modules/menu/gp-menu-button-applet.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/menu/gp-menu-button-applet.c b/modules/menu/gp-menu-button-applet.c
index b864b9c8a..0eaac3733 100644
--- a/modules/menu/gp-menu-button-applet.c
+++ b/modules/menu/gp-menu-button-applet.c
@@ -418,7 +418,7 @@ settings_changed_cb (GSettings          *settings,
       gboolean show_arrow;
 
       show_arrow = g_settings_get_boolean (priv->settings, "show-arrow");
-      gtk_widget_set_visible (priv->image, show_arrow);
+      gtk_widget_set_visible (priv->arrow, show_arrow);
     }
   else if (g_strcmp0 (key, "tooltip") == 0)
     {
-- 
2.43.0

